problem_builder/mentoring.py

```diff
@@ -174,11 +175,11 @@ def get_options(self):
             return xblock_settings[self.options_key]
         return _default_options_config
 
-    def get_option(self, option):
+    def get_option(self, option, default=False):
```

Comment:
NIT: I'd rather either have a default of ``None`` or no default for default parameter. ``None`` is a sensible default for every type of settings parameter, while ``False`` only sensible for bools. 

From these two options I'd rather have no default, and have the default specified when parameter is read.